lib/shouter.rb

```diff
@@ -0,0 +1,14 @@
+class Shouter
+
+  def self.setup
+    @@shouts = []
+  end
+
+  def self.all
+    @@shouts
+  end
+
+  def self.shout(shout_content, author = 'Solo')
+    @@shouts.unshift([author, Time.now, shout_content])
+  end
+end
```

Comment:
This is more a reflection on my code, but I'll be kicking myself if the solution was actually this simple. Perhaps I overcomplicated.